### ISSUE TITLE
"More Reports",  "Search", and "Super Search" links missing

### ISSUE THREAD

**Issue by t-matsuu** [thundernest/crash-diverter#9]
Like Mozilla Crash Reports.
![image](https://user-images.githubusercontent.com/21326714/152999624-354f6570-e795-43aa-8cb1-9efcb613b424.png)

**Comment by t-matsuu** [comment]
"Super Search" link is also missing.
![image](https://user-images.githubusercontent.com/21326714/153000780-4d7826e4-4ef5-4a0b-8f21-d4735996e7ed.png)

**Comment by Sancus** [comment]
Just to be clear, it's not very likely the Thunderbird crash reports site will ever have most of the functionality of the Firefox one, unless we return to using Socorro at some point in the future. I'm not going to close this, but I have no plans to add any of this stuff at this time.

**Comment by t-matsuu** [comment]
> Just to be clear, it's not very likely the Thunderbird crash reports site will ever have most of the functionality of the Firefox one, unless we return to using Socorro at some point in the future. I'm not going to close this, but I have no plans to add any of this stuff at this time.

I see.

Does this mean below?
* Only authorized Thunderbird developers can see the trends of crash stats after moving crash stats website to crash-stats.thunderbird.net.
* Non authorized ones can only access crash stats who know the Breakpad ID.

**Comment by Sancus** [comment]
> * Only authorized Thunderbird developers can see the trends of crash stats after moving crash stats website to crash-stats.thunderbird.net.
> 
> * Non authorized ones can only access crash stats who know the Breakpad ID.

Yes. Unfortunately, the 3rd party service we are using, Backtrace, provides limited access to data through the API. And we can't give out Backtrace accounts to anyone as they are expensive and also provide full access to all crash data, including potentially private data.

I would like to get more data on crash-stats.thunderbird.net, but it won't be soon.

It is also possible that we may return to Socorro in the future(1-2 years from now). We're definitely aware this situation is not ideal.
